code/modules/admin/verbs/adminpm.dm

```diff
@@ -373,6 +382,13 @@
 		show_ui(usr)
 		return
 
+	if(href_list["altclose"])
+		message_admins("[key_name_admin(usr)] closed a force-opened PM window")
```

Comment:
I'd make it only print if they don't reply before closing it. It's a bit spammy otherwise.

Reply:
Disagree. This close is only used when someone closes a window which we forced open on someone, which rarely happens (Only when we need urgent responses).

Reply:
Why not just make it generate a popup for confirmation, like "An admin opened this window because they want your answer. Are you sure you want to close without answering? This may get you banned.".

Reply:
@Kyep the whole point of this is so the admin knows the player intentionally closed the window, not the player closing it. If a player is on a grief spree, they wont care if they see a warning thats like "This may get you banned". The entire point of this is to alert admins to closures, and remove the "I didnt see the PM window" card, as well people saying "BYOND must have bugged because it never opened"